packages/expo-dev-menu/ios/DevMenuManager.swift

```diff
@@ -314,14 +297,15 @@ open class DevMenuManager: NSObject, DevMenuManagerProtocol {
     if isVisible == visible {
       return false
     }
-    return delegate?.devMenuManager?(self, canChangeVisibility: visible) ?? true
+    
+    return true
   }
 
   /**
    Returns bool value whether the onboarding view should be displayed by the dev menu view.
    */
   func shouldShowOnboarding() -> Bool {
-    return delegate?.shouldShowOnboarding?(manager: self) ?? !DevMenuSettings.isOnboardingFinished
+    return !DevMenuSettings.isOnboardingFinished
```

Comment:
I don't think we have an onboarding popup anymore - should we consider putting one back?

Reply:
Did you remove the onboarding popup? If yes, I think we should consider putting it back. It's an introduction to dev-menu basic functionality.